docs/api/python/ndarray/ndarray.md

```diff
@@ -131,6 +131,7 @@ The `ndarray` package provides several classes:
     NDArray.flatten
     NDArray.expand_dims
     NDArray.split
+    NDArray.diag
 ```
```

Comment:
Sorry I didn't make it clear - there're two places to add per file. For `ndarray.md`, One is NDArray.diag (fluent method) and the other is (ndarray.)diag at line 360.

Reply:
Done